core/src/main/java/org/jruby/RubyBignum.java

```diff
@@ -236,6 +236,63 @@ public static BigInteger long2big(long arg) {
      *  ================
      */
 
+    /** rb_big_ceil
+     *
+     */
+    @Override
+    public IRubyObject ceil(ThreadContext context, IRubyObject args){
```

Comment:
`args` should be named `arg` or something (everywhere) - otherwise its a bit confusing.